lib/api/src/modules/shortcuts.ts

```diff
@@ -251,7 +254,14 @@ export default function initShortcuts({ store }: Module) {
           fullApi.navigate('/settings/shortcuts');
           break;
         }
-
+        case 'collapseAll': {
```

Comment:
Can we use the enums for these?

Reply:
yes, it would be cleaner, but didn't want to unsettle the rest of the code. I can add it to this PR, or maybe create a new one?